docs/io/fits/index.rst

```diff
@@ -178,6 +178,22 @@ When writing a file (e.g., with the :func:`writeto` function), compression will
 be determined based on the filename extension given, or the compression used in
 a pre-existing file that is being written to.
 
+
+Working with non-standard files
+"""""""""""""""""""""""""""""""
+`astropy.io.fits` adheres to the FITS standard and verifies that all fields within a FITS file conform
+to the standard. It may raise ``VerifyError`` if a field is deemed to be non-compliant. This action can be modified
+using method ``HDUList.verify``. It is typically invoked after opening the file but before reading any headers or data::
+
+    >>> hdul = fits.open(fits_image_filename)
+    >>> hdul.verify('silentfix+warn')
+    >>> data = hdul.data
```

Comment:
Please use the context manager here, `with fits.open(fits_image_filename) as hdul: ...`. Also for the educational purpose of the example I would prefer `fix` instead of `silentfix` ;).

Reply:
OK. Sounds good to me.  I was following the format of earlier examples on the documentation page which do not use the "with" statement. I'm ok with 'fix'